packages/elements/src/utils/http-spec/IServer.ts

```diff
@@ -3,6 +3,10 @@ import { IServer } from '@stoplight/types';
 export const getServerUrlWithDefaultValues = (server: IServer): string => {
   let url = server.url;
 
+  if (url[0] === '/' && typeof window !== 'undefined') {
```

Comment:
We may also want to do that when there is no `/`. In such case it also shows nothing in `API Base URL` part and breaks RequestSamples. Unless we want that to happen for these kind of inputs.